aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,10 +1,177 @@
-﻿.. include:: /../common/stub-topic.txt
+Bundling and Minification
+=========================
 
-.. _bundling-minification-index:
+By `Rick Anderson`_ and `Erik Reitan`_ 
 
-|stub-icon| Bundling and Minification
-=====================================
+Bundling and minification are two techniques you can use in ASP.NET 5 to improve page load performance for your web application. Bundling makes it easy to combine or bundle multiple files into a single file. Minification performs a variety of different code optimizations to scripts and CSS, which results in smaller payloads. Used together, bundling and minification improves load time performance by reducing the number of requests to the server and reducing the size of the requested assets (such as CSS and JavaScript files).
 
-.. include:: /../common/stub-notice.txt
+This article explains the benefits of using bundling and minification, including how these features are used in the **ASP.NET 5 Web Application** template of `Visual Studio 2015 <http://go.microsoft.com/fwlink/?LinkId=517106>`_. 
 
-.. _issue: https://github.com/aspnet/Docs/issues/101
\ No newline at end of file
+.. contents:: In this article:
+  :local:
+  :depth: 1
+
+Overview
```

Comment:
I think your initial overview paragraph above is great and probably sufficient. This overview section feels a little two detailed and has some content that is probably point-in-time (ex number of simultaneous connections used by browsers today). I think you could probably pull this whole section and just jump to the Bundling section